### DISCUSSION TITLE
What's the difference in the slim version?

### DISCUSSION THREAD

**Discussion by Luc45** [picocss/pico#369] [Help] [answered]
What is removed in the Slim version that makes it smaller?

Thanks.

**Comment by nakibrayane** [comment, marked as answer]
these are the only moduels avalable in the slim version of pico css.

- variables

- themes/default


- layout/document (html)
- layout/sectioning (body, header, main, footer)
- layout/container (.container, .container-fluid)
- layout/section (section)
- layout/grid (.grid)
- ayout/scroller (figure)

- content/typography (a, headings, p, ul, blockquote, ...)
- content/embedded (audio, canvas, iframe, img, svg, video)
- content/button (button, a[role=button], type=button, type=submit ...)
- content/form (input, select, textarea, label, fieldset, legend)
- content/table (table, tr, td, ...)

- utilities/accessibility (-ms-touch-action, aria-*)
- utilities/reduce-motion (prefers-reduced-motion)
